server/plugins/schema-plugin-api/src/main/java/com/oceanbase/odc/plugin/schema/api/TableExtensionPoint.java

```diff
@@ -43,4 +43,6 @@ public interface TableExtensionPoint extends ExtensionPoint {
     String generateCreateDDL(Connection connection, DBTable table);
 
     String generateUpdateDDL(Connection connection, DBTable oldTable, DBTable newTable);
+
+    String generateUpdateDDLWithoutRenaming(Connection connection, DBTable oldTable, DBTable newTable);
```

Comment:
why should we create this method?

Reply:
because `DBTableService#generateUpdateDDLWithoutRenaming` call this method，but it is only used in ShadowTableComparingTask at present.
However, considering that the schema-plugin is only for ODC to view database objects, it is indeed unnecessary to add this method here, so I would delete this method.

Reply:
we may move `DBTableService#generateUpdateDDLWithoutRenaming` to `ShadowTableComparingTask`, which is only used by `ShadowTableComparingTask`.

Reply:
👌，`DBTableService#listTables` and `DBTableService#isLowerCaseTableName` are also only used in `ShadowTableComparingTask`，but will not move them to `ShadowTableComparingTask` because these method can be commonly used in the future by other method.